tests/integration/test_cli.py

```diff
@@ -93,14 +93,39 @@ def test_multi_reference_is_duplicated(tmpdir):
         autohint(inpaths + ['-o'] + outpaths + ['-r', reference])
 
 
+tx_found = False
+try:
+    subprocess.call(["tx", "-v"])
+    tx_found = True
+except OSError:
```

Comment:
I think `subprocess.CalledProcessError` should be included here as well, in case `tx` is found but does not return `0`. So something like this,
```python
except (subprocess.CalledProcessError, OSError):
```

Reply:
I was reluctant to do this, since many AFDKO tools (used to?) return non-0 for help options. But lets do it and see.